app/controllers/sessions_controller.rb

```diff
@@ -4,9 +4,15 @@ def new; end
   def create
     @user = User.find_by email: params[:session][:email].downcase
     if @user&.authenticate(params[:session][:password])
+      if @user.activated
       log_in @user
-      params[:session][:remember_me] == "1" ? remember(@user) : forget(@user)
+      params[:session][:remember_me] == "1" ? remember(user) : forget(user)
       redirect_back_or @user
```

Comment:
indent, please

Reply:
Please indent your code like in this picture 
 
![2021-09-06_10-58](https://user-images.githubusercontent.com/72131384/132158785-4a4c1318-af3d-46dd-b33f-ebab8dbb6a11.png)